[MOS-281] Base documentation update (#3645)
mudita/MuditaOS: doc/download_assets.md

> e Download Assets

Author: fixed tx :)

To use GitHub API your GitHub login and special token (API password) are required.
This token can by generated on "User" -> [Settings](https://github.com/settings/profile) page in section [Developer settings](https://github.com/settings/apps) -> [Personal acces tokens](https://github.com/settings/tokens) section.

> It has to have at least read access to repositories

Reviewer: ```suggestion
It has to have at least read access to repositories.
```

For more info please refer to the Github help page about adding tokens:
[github documentation: Creating a personal access token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token)

### Scopes
When you generate a token you can set "Scopes" for which the token will work.
For downloading assets, we need a ":repo" scope with all sub-scopes.

**Note:** Tokens are displayed only during their creation, so this is the only moment you can copy them.
If you forget to do this, you will have to generate a new token.

> **Note:** With token added [please double check](#Testing-for-proper-asset-download-configuration) it's working before issuing not working toolkit as most of the problems lies with no, or improper configuration.

Reviewer: ```suggestion
**Note:** With token added [please double check](#Testing-for-proper-asset-download-configuration), if it's working before issuing not working toolkit as most of the problems lies with no, or improper configuration.
```

## Storing settings

The `download_assets.py` loads authorisation data via git configuration.

> The process requires so that this data isstored in your current repository or global git configuration.

Reviewer 2: ```suggestion
The process requires that this data is stored in your current repository or global git configuration.
```
